.swcrc.build.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Just leaving a comment here for future reference.

When we change the package to use SWC for other parts of the stack, such as testing, we need to keep in mind to use a different `.swcrc` file. This one is explicitly ignoring test files and such, so will not work with Jest.